x-pack/plugins/session_view/common/types/process_tree/index.ts

```diff
@@ -79,6 +79,7 @@ export interface ProcessFields {
 export interface ProcessSelf extends Omit<ProcessFields, 'user'> {
   parent: ProcessFields;
   session_leader: ProcessFields;
+  session: ProcessFields;
```

Comment:
Maybe it's good to indicate that replacing the usages of `session_leader` with `session` is only a temporary implementation

Reply:
good catch, I'll add that